sdk/resourcemanager/Azure.ResourceManager/src/autorest.md

```diff
@@ -659,7 +659,7 @@ directive:
       $['x-ms-client-name'] = "ResourceType"
   - rename-model:
       from: CheckNameAvailabilityRequest
-      to: ManagementGroupNameAvailabilityOptions
```

Comment:
@m-nash  please confirm if we should rename it

Reply:
already revert the change